package.json

```diff
@@ -59,20 +84,22 @@
     "responsive"
   ],
   "dependencies": {
-    "babel-plugin-inline-package-json": "^2.0.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.16.0",
+    "@embroider/macros": "^0.47.2",
     "broccoli-funnel": "^2.0.1",
     "broccoli-merge-trees": "^2.0.0",
-    "ember-cli-babel": "^6.6.0",
-    "ember-cli-htmlbars": "^2.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
-    "ember-get-config": "0.2.4",
+    "ember-cli-babel": "^7.26.6",
+    "ember-cli-htmlbars": "^5.7.1",
+    "ember-get-config": "0.2.4 - 0.5.0",
     "ember-resize-aware": "1.2.0",
     "fastboot-transform": "^0.1.2",
     "imgix-core-js": "2.3.2",
```

Comment:
In a separate PR we can switch this to use `imgix/js-core` since thats the new preferred name